screens/products-list/products.css

```diff
@@ -0,0 +1,131 @@
+body {
+    background-color: var(--primary-light);
+}
+
+.product-container {
+    display: flex;
+    gap: 1rem;
+    height: auto;
+    margin: 1rem;
+}
+
+.product-filters {
+    display: flex;
+    flex-direction: column;
+    gap: 1.3rem;
+    border: 3px solid var(--primary-light);
+    border-radius: 5px;
+    width: 20%;
+    padding: 1rem;
+}
+
+.product-filters div h4 {
+    font-weight: 600;
+}
+
+.filter-category input {
+    width: 17px;
+    height: 17px;
+}
+
+.filter-rating input,
+.filter-sort input {
+    width: 19px;
+    height: 19px;
+}
+
+.products-all {
+    width: 80%;
+    height: auto;
+    border-radius: 5px;
+    padding: 0 0.5rem;
+}
+
+.products-all h2 {
+    font-weight: 500;
+}
+
+.filter-title {
+    display: flex;
+    justify-content: space-between;
+}
+
+/* Slider  */
+.rangeValue {
+    font-size: 0.8rem;
+    color: var(--primary-color);
+    font-weight: normal;
+}
+
+.slider-container {
+    margin: 0;
+    padding: 0 0.4rem;
+    border: 1px solid var(--primary-color);
+    border-radius: 5px;
+}
+
+.slider-two {
+    width: 15vw;
+    height: 10px;
+    -webkit-appearance: none;
+    background: var(--primary-color);
+    outline: none;
+    border-radius: 15px;
+    overflow: hidden;
+}
+
+.slider-two::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    width: 0.8rem;
+    height: 1rem;
+    border-radius: 50%;
+    background: var(--primary-light);
+    cursor: pointer;
+    border: 4px solid var(--primary-dark);
+    box-shadow: 407px 0 0 400px var(--primary-light);
+}
+
+/* Products page  */
+.showing-product-result {
+    font-weight: normal;
+    font-size: 0.7rem;
+}
+
+.total-products-container {
+    display: flex;
+    gap: 2rem;
+    flex-wrap: wrap;
+}
+
+.card-container {
+    width: 22%;
+    padding: 0.4rem;
+}
+
+/* card badge  */
+.card-badge::after {
+    width: 30%;
+    padding: 0.2rem 1rem;
+    font-weight: 500;
+}
+
+/* Card details container */
+.card-product-brand,
+.card-product-name,
+.card-product-price,
+.card-product-offer {
+    font-weight: 500;
+}
+
+/* Add to cart btn  */
+.btn-dark {
+    margin: 0.5rem 0 0 0;
```

Comment:
You can use `margin-top: 0.5rem;` here.

Reply:
I think, if she just use margin-top, the other 3 sides might get overwritten, explicitly making them 0 as per requirement is fine.

Reply:
That's also a valid point.

Reply:
Thank you @jeetsdev @abhisekgupta001 :)